packaging/debian/install

```diff
@@ -4,4 +4,5 @@ etc/rsyslog.d/*
 etc/sysctl.d/*
 lib/udev/rules.d/*
 lib/udev/*
+lib/dracut/*
```

Comment:
Debian doesn't use dracut so please remove this.

Reply:
Done. Thanks for checking it @zmarano